resources/sdks.md

```diff
@@ -1,13 +1,9 @@
 # Official SDKs / Libraries
 
-You can use MeiliSearch API wrappers in your favorite language. These libraries support the main methods for search and indexation and are tested against MeiliSearch v0.9.0.
+You can use MeiliSearch API wrappers in your favorite language. These libraries support the main methods for searching and indexing documents and are tested against MeiliSearch v0.9.0.
```

Comment:
Outdated information hehe. Plus I removed the testing part. Not sure if that's useful information
```suggestion
You can use MeiliSearch API wrappers in your favorite language. These libraries support all API routes.
```